### PR TITLE
Add Sammy and change title colour for better contrast

### PR DESCRIPTION
**Pull request by gracelu0**
- add sammy to execs
- change about title 'Enter StormHacks' to darker orange colour for better contrast

### CODE REVIEW

File: src/pages/About/About.module.scss [hunk 1]
```diff
@@ -32,7 +32,7 @@
 
       .title {
         @include medium(48px, Poppins, 24px);
-        color: #ea8033;
+        color: #F06500;
```
Comment (praneershrest) [inline src/pages/About/About.module.scss]: check this, I think Nathan changed the orange theme. If that's the case, make the change in the index.scss file

**Comment by joshpeng1999** [comment]
Are we Merging into main?

**Review by joshpeng1999 — APPROVED** [review]

**Comment by praneershrest** [comment]
should be merging to dev first!

File: src/pages/About/About.module.scss (hunk 1 above)
Comment (gracelu0) [reply src/pages/About/About.module.scss]: the orange in index.scss was correct, updated to use `$orange`

**Review by praneershrest — APPROVED** [review]

**Pull request merged**